welcome/index.adoc

```diff
@@ -42,10 +42,14 @@ You can also deploy a cluster on AWS infrastructure that you provisioned yoursel
 
 - **xref:../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[Install a cluster on Azure]**: You can deploy clusters with xref:../installing/installing_azure/installing-azure-default.adoc#installing-azure-default[default settings], xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-customizations[custom Azure settings], or xref:../installing/installing_azure/installing-azure-network-customizations.adoc#installing-azure-network-customizations[custom networking settings] in Microsoft Azure. You can also provision {product-title} into an xref:../installing/installing_azure/installing-azure-vnet.adoc#installing-azure-vnet[Azure Virtual Network] or use xref:../installing/installing_azure/installing-azure-user-infra.adoc#installing-azure-user-infra[Azure Resource Manager Templates] to provision your own infrastructure.
 
+- **xref:../installing/installing_azure_stack_hub/preparing-to-install-on-azure-stack-hub.adoc#preparing-to-install-on-azure-stack-hub[Install a cluster on Azure Stack Hub]**: You can install {product-title}on Azure Stack Hub on user-provisioned infrastructure.
```

Comment:
@codyhoag, are you happy with these links, or do you want them to point to a different entry point?